common/notification/index.ts

```diff
@@ -243,6 +246,21 @@ async function createConcreteNotification(
     return concreteNotification;
 }
 
+const getNotificationChannelPreferences = async (user: User, concreteNotification: ConcreteNotification): Promise<Channels> => {
+    const { messageType } = getMessage(concreteNotification);
```

Comment:
So the message type now determines the notification channel? I don't like that. This should be notification.type.

Reply:
currently we only map types in templates. as we are moving templates to db anyway we could add the type field to notification table. would that be a good solution

Reply:
Yes, can also be done as part of LF-60 if you want to bring this PR in

Reply:
so I will keep this solution till change it in LF-60, okay?